panelstudio-mc19/src/main/java/com/lukflug/panelstudio/mc17/package-info.java

```diff
@@ -0,0 +1,6 @@
+/**
+ * PanelStudio-MC utility functions designed for FabricMC 1.17.x and 1.18.x.
+ * This library only depends on the Fabric Yarn mappings and not on the Fabric Mod Loader or the Fabric API.
+ * @author lukflug
+ */
```

Comment:
Update description. Plus, add yourself as the author of this port. As well as in the classes you modified.